api/src/main/java/jakarta/faces/view/facelets/TextHandler.java

```diff
@@ -24,7 +24,7 @@
  *
  * @since 2.0
  */
-public interface TextHandler {
+public interface TextHandler extends FaceletHandler {
```

Comment:
If this is a backwards imcompatible change should we document this as it appears it is. Anyone delivering a TextHandler in Faces 4 will have to now implement whatever FaceletHandler exposes. Right?

Reply:
Indeed, we need to document this somewhere in spec.

Reply:
We could add a default method to TextHandler to provide something for the one method FaceletHandler defines?

Reply:
Can also. Hereby done: https://github.com/jakartaee/faces/pull/2018/commits/38fe4cd44abb6285b4384ee2e87286be90577282